src/frontend/components/StudentLiveWrapper/index.spec.tsx

```diff
@@ -0,0 +1,302 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import {
+  LivePanelItem,
+  useLivePanelState,
+} from 'data/stores/useLivePanelState';
+import { liveState } from 'types/tracks';
+import { videoMockFactory } from 'utils/tests/factories';
+
+import { LiveVideoWrapper } from '.';
+
+const mockVideo = videoMockFactory();
+jest.mock('data/appData', () => ({
+  appData: {
+    video: mockVideo,
+  },
+  getDecodedJwt: () => ({
+    permissions: {
+      can_update: false,
+    },
+  }),
+}));
+jest.mock(
+  'components/VideoPlayer',
+  () =>
+    ({ playerType }: { playerType: string }) =>
+      <span>video {playerType}</span>,
+);
+jest.mock('components/DashboardVideoLiveJitsi', () => () => (
+  <span>dashboard video live jitsi</span>
+));
+jest.mock('components/LiveVideoPanel', () => ({
+  LiveVideoPanel: () => <span>live video panel</span>,
+}));
+jest.mock('components/StudentLiveControlBar', () => ({
+  StudentLiveControlBar: () => <span>student live control bar</span>,
+}));
+jest.mock('components/StudentLiveInfoBar', () => ({
+  StudentLiveInfoBar: () => <span>student live info bar</span>,
+}));
+jest.mock('./StudentLiveLayout', () => ({
+  StudentLiveLayout: ({
+    actionsElement,
+    isPanelOpen,
+    liveTitleElement,
+    mainElement,
+    sideElement,
+  }: {
+    actionsElement: React.ReactElement;
+    isPanelOpen?: boolean;
+    liveTitleElement: React.ReactElement;
+    mainElement: React.ReactElement;
+    sideElement?: React.ReactElement;
+  }) => (
+    <div>
+      {actionsElement}
+      <span>
+        is panel open{' '}
+        {isPanelOpen === undefined
+          ? 'undefined'
+          : isPanelOpen
+          ? 'true'
+          : 'false'}
```

Comment:
```suggestion
        is panel open `${isPanelOpen}`
```
I'm not sure if it works in this context…

Reply:
40c6f96

Thanks, à forgot about this (in the first place I did as you suggest but this is not working, I went with the ternary just temporary but ...)